console/v1/0000_10_consoleplugin.crd.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consoleplugins.console.openshift.io
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    displayName: ConsolePlugin
+    description: Extension for configuring openshift web console plugins.
+spec:
+  scope: Cluster
+  group: console.openshift.io
+  preserveUnknownFields: false
+  versions:
+  - name: v1
```

Comment:
Thinking if we should not put this to `v1alpha1`, in case we decide to change any fields for 4.8
@spadgett thoughts ?

Reply:
We did say it would be alpha in the enhancement since it's only meant for internal use at this time.